packages/protojs/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nibiruchain/protojs",
   "description": "Protobuf-generated types for Nibiru Chain",
-  "version": "0.19.1",
+  "version": "0.20.0",
```

Comment:
Let's try to keep the package minor version the same as `nibiru` [versions](https://github.com/NibiruChain/nibiru/releases)